gensrc/thrift/Partitions.thrift

```diff
@@ -45,14 +45,10 @@ enum TPartitionType {
 }
 
 enum TDistributionType {
-  UNPARTITIONED,
-
-  // round-robin partition
-  RANDOM,
-
-  // unordered partition on a set of exprs
-  // (partition bounds overlap)
-  HASH_PARTITIONED
+  ANY,
```

Comment:
This change is safe?

Reply:
This type is unused, and it seems out of date, I make it equals with DistributionSpec::DistributionType in DistributionSpec.java